exonum/src/blockchain/service.rs

```diff
@@ -169,9 +169,14 @@ pub trait Service: Send + Sync + 'static {
     /// [the `Service` example above](#examples).
     fn tx_from_raw(&self, raw: RawTransaction) -> Result<Box<dyn Transaction>, MessageError>;
 
-    /// Initializes the information schema of the service
-    /// and generates an initial service configuration.
-    /// This method is called on genesis block creation.
+    /// This method is invoked for all deployed services during the blockchain initialization
+    /// on genesis block creation.
+    /// During the handling of the method the service able to make 2 things:
+    /// - store its own initial state to the storage (Fork)
+    /// - return an initial [global configuration][doc:global_cfg] of the service in the JSON
+    /// format, in case service has global configuration parameters. This configuration used
+    /// to create a genesis block.
```

Comment:
It's not clear, at what moment this global configuration is used in terms of service configuration process (TEMPLATE -> CONFIG -> FINALIZE -> RUN). Moreover, it's not clear what happens with parameters already specified by user? Will parameter be overwritten by this method?